TcUnit-Runner/Program.cs

```diff
@@ -304,6 +329,31 @@ static void Main(string[] args)
             {
                 System.Threading.Thread.Sleep(10000);
 
+                /* Monitor the AdsState for each PLC that is used in the
+                 * solution. If we can't connect to the Ads Router, we just
+                 * carry on.
+                 */
+                try
+                {
+                    foreach (int amsPort in AmsPorts)
+                    {
+                        tcAdsClient.Connect(AmsNetId, amsPort);
```

Comment:
I don't have so much experience with this, but how does this AdsClient work. I mean, I guess one assumption is that the machine where this is running needs to have an active AMS connection to the target? Is there a DLL for ADS that follows along TcUnit-Runner... ?